src/contents/ui/Compact.qml

```diff
@@ -51,7 +53,43 @@ Item {
     Rectangle {
         anchors.fill: parent
         color: backgroundColor
-        radius: plasmoid.configuration.panelBackgroundRadius
+        radius: compact.panelBackgroundRadius
+        Item {
+            // start the progress after the cover/thumbnail if it is as tall as the panel thickness
+            property int panelIconSpace: panelIcon.type === PanelIcon.Type.Image && (compact.iconSize === compact.widgetThickness) ? compact.iconSize - Math.min(plasmoid.configuration.albumCoverRadius, compact.iconSize / 2) : 0
```

Comment:
This works fine with square images, but with non-square images it appears like this: 

<img width="393" height="52" alt="Image" src="https://github.com/user-attachments/assets/52438ab6-b719-461b-9ecd-7a71661c70e2" />

Maybe it's better to always start the progress bar from x: 0, regardless of whether it's visible when iconSize == widgetThickness. What do you think?